docs/user/metrics/memory_distribution.md

```diff
@@ -9,6 +9,8 @@ That is, the function from a value \\( x \\) to a bucket index is:
 
 This makes them suitable for measuring memory sizes on a number of different scales without any configuration.
 
+> **Note** Check out how this bucketing algorithm would behave on our [Histogram Simulator](../../appendix/histograms.html)
```

Comment:
We can add params to this URL to open the simulator on a plot that matches the bucketing algorithm for the memory distribution, but I honestly was not sure what these params were. We would need, lowerBound, upperBound and bucketCount.

Reply:
Appreciate the work here.
I think the whole point though is that the bucketing algorithm for memory and timing distributions  is significantly different from the old one used in Firefox Telemetry (and custom distributions).
The bounds are not configurable for example.

Thus we should hard-code these as the defaults on top of the linear/exponential histograms (that are available as custom distributions)

Reply:
That was what I figured, but was not sure. Okay, I'll send a new commit adding these two to the simulator.